fonduer/meta.py

```diff
@@ -17,7 +17,12 @@ def new_sessionmaker():
     # Otherwise any e.g. query starts a transaction, locking tables... very
     # bad for e.g. multiple notebooks open, multiple processes, etc.
     if Meta.postgres and Meta.ready:
-        engine = create_engine(Meta.conn_string, isolation_level="AUTOCOMMIT")
+        engine = create_engine(
+            Meta.conn_string,
+            client_encoding="utf8",
+            use_batch_mode=True,
```

Comment:
do you still need this?

Reply:
It's not currently utilized since we're not using the [bulk execution functions](http://docs.sqlalchemy.org/en/latest/faq/performance.html#i-m-inserting-400-000-rows-with-the-orm-and-it-s-really-slow) from sqlalchemy. But, having it hear leaves that option open in the future.